src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -200,17 +196,45 @@ static void CheckForExperimentStatus(ITextView textView, Document document)
             return true;
         }
 
-        public async Task<AsyncCompletionData.CompletionContext> GetCompletionContextAsync(
+        public Task<AsyncCompletionData.CompletionContext> GetCompletionContextAsync(
             IAsyncCompletionSession session,
             AsyncCompletionData.CompletionTrigger trigger,
             SnapshotPoint triggerLocation,
             SnapshotSpan applicableToSpan,
             CancellationToken cancellationToken)
+        {
+            return GetCompletionContextWorkerAsync(session, trigger, triggerLocation, applicableToSpan, isExpanded: false, cancellationToken);
+        }
+
+        public async Task<AsyncCompletionData.CompletionContext> GetExpandedCompletionContextAsync(
+            IAsyncCompletionSession session,
+            AsyncCompletionData.CompletionExpander expander,
+            AsyncCompletionData.CompletionTrigger intialTrigger,
+            SnapshotSpan applicableToSpan,
+            CancellationToken cancellationToken)
+        {
+            Debug.Assert((object)expander == FilterSet.Expander);
```

Comment:
Document reason for checking the equality here. If you want to produce extra items exclusively as reaction to Roslyn's expander, then return CompletionContext.Empty when the expander doesn't match.

Reply:
Thanks! Fixed.